agent/src/test/java/com/code_intelligence/jazzer/runtime/BUILD.bazel

```diff
@@ -10,3 +12,16 @@ java_test(
         "@maven//:junit_junit",
     ],
 )
+
+java_test(
+    name = "TraceCmpHooksTest",
+    srcs = [
+        "TraceCmpHooksTest.java",
+    ],
+    target_compatible_with = SKIP_ON_WINDOWS,
```

Comment:
Is this needed because of the mock driver?

Reply:
Yes, my understanding is that the mock driver is not properly running on windows. Did I get that wrong?
The test failed on windows only.

Reply:
You are right. Let's keep this as is and track the proper fix to that problem in #376.